makefiles/docker.inc.mk

```diff
@@ -1,6 +1,10 @@
 export DOCKER_IMAGE ?= riot/riotbuild:latest
 export DOCKER_BUILD_ROOT ?= /data/riotbuild
 export DOCKER_FLAGS ?= --rm
+ifdef BAUD
+export DOCKER_FLAGS += -e BAUD=$(BAUD)
```

Comment:
Isn't ```BAUD``` exported, and thus ```-e BAUD``` sufficient?

Reply:
well I am not sure whether it will be defined inside docker even if there is not externaly

Reply:
If it is exported by make, it will be part of the environment of the "docker run" call. In that case, just specifying the ```-e <name>``` will make docker copy the current value.

Reply:
This means we should rather do a `export BAUD` inside the Makefile then?

Reply:
> This means we should rather do a export BAUD inside the Makefile then?

Add it to ```makefiles/vars.inc.mk```.